about.html

```diff
@@ -0,0 +1,88 @@
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <div id="modal"class="modal mobile">
+    <img alt="modal" class="modal-close" src="Images/Icons/x-mark-black-symbol-and-white-symmetry.jpg">
+    <ul class="ul-modal">
+        <li class="il-modal"><a href="index.html">Home</a></li>
+        <li class="il-modal"><a href="index.html">Results</a></li>
+        <li class="il-modal"><a href="index.html">Awards</a></li>
+        <li class="il-modal"><a href="index.html">News</a></li>
+        <li class="il-modal"><a href="index.html">Teams</a></li>
+      </ul>
+  </div>
+  <section class="section-main-about">
+    <span id="home-section-span-title" class="about-section-span mobile">FIFA World Cup</span>
+    <h1 class="about-section-h1 bold h1-about">FIFA World Cup Qatar 2022</h1>
+    <div class="home-section-div about-main-div">
+      <span class="">The 2022 FIFA World Cup was held in Qatar from November 21st to December 18th, 2022. It featured 32 teams playing in eight stadiums across five host cities, with the final held at the newly completed Lusail Iconic Stadium. The tournament aimed to promote football in the Middle East and bring people together in a celebration of sport and culture.</span>
+    </div>
+    <span  class="home-section-span section-main-about-span">Please contact us per Email for any further questions
+        <br> about FIFA Worls Cup 2022 </span>
+    <span class="home-section-span section-main-about-span2 bold">media@fifa.org</span>
+  </section>
+  <section class="logo-section">
+    <h2 class="logo-section-h2">FIFA World Cup 2022 Logo</h2>
+    <hr class="section-hr">
+    <p class="logo-section-p"> The FIFA World Cup 2022 logo was inspired by the traditional woollen shawls worn during the winter months in Qatar. <span class="desktop logo-section-p"><br>It features a design that represents the infinity symbol and the number eight, which is a reference to the eight stadiums that hosted matches. <br> The logo is intended to embody the tournament's theme of 'Unity' and symbolize the connections between the event, the host nation, and the world.</span> </p>
+    <div class="logo-section-div"><img alt="Logo" src='Images/logo.png'></div>
+  </section>
+  <section class='past-events'>
+    <h2>See the Past FIFA World Cups</h2>
+    <hr class="section-hr">
+    <div class="past-events-container">
+      <div class="past-events-img-container event-before"> <span class="bold">2018</span><br>
+        <span class="past-events-img-container-span">FIFA World Cup 2018 Russia</span></div>
+    <div class="past-events-img-container2 event-before"> <span class="bold">2014</span><br>
+        <span class="past-events-img-container-span">FIFA World Cup 2014 Brazil</span></div>
+    </div>
+    
+  </section>
+  <footer class="about-footer footer-2">
+    <div class="footer-logo logo-2"></div> <p class="footer-p footer-p2"> FIFA World Cup  <br> FIFA World Cup Qatar 2022</p>
+  </footer>
```

Comment:
Great job @hafiz1379 🤝

- [ ] [Optional] Kindly ensure your design matches the design given. You may make sure your footer section for your mobile screen matches the design given.
- Your design
![Your-about-footer](https://user-images.githubusercontent.com/63917725/234539355-548d04b5-0efa-42be-8b12-0f850a5bf5b4.PNG)
- Given design
![Given-about-footer](https://user-images.githubusercontent.com/63917725/234539459-de46667a-7a1b-47e9-afb9-6e120b5f7680.PNG)